src/malik/socnet.js

```diff
@@ -1 +1,86 @@
+//Log your name
 console.log("Abdulmalik Yusuf");
+
+
+/*Activity One (data types)
+=========================
+
+14 === Number
+-80 === Number
+8.5 === Number
+true === Boolean
+(5 < 17) === Boolean
+null === Undefined
+3.142 === Number
+'Pi' === String
+67+18 === Number
+*/
+
+// Code that converts Integers into Binary numbers
+var convertToBinary = function(number) {
+  var quotient = number;
+  var remainder = [];
+
+  do {
+    remainder.push (quotient % 2);
+    quotient = Math.floor(quotient / 2);
+    console.log(quotient);
+  }
+  while (quotient >= 1) {
+    console.log(remainder.reverse().join(''));
+  }
+}
+
+var x = 20;
+convertToBinary();
+console.log(x.toString(2));
+
+
+//Code to print student grades
+var printGrade = function(x) {;
```

Comment:
It's usually recommended to use parameter names that describe what the parameter represents.